app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/JourneyConfig.scala

```diff
@@ -16,7 +16,8 @@
 
 package uk.gov.hmrc.incorporatedentityidentificationfrontend.models
 
-import play.api.libs.json.{JsObject, JsResult, JsValue, Json, OFormat}
+import play.api.libs.json._
+import uk.gov.hmrc.incorporatedentityidentificationfrontend.testonly.forms.TestCreateJourneyForm.signOutUrl
```

Comment:
This shouldn't be imported, you need to add sign out link to the read and writes